back/engines/commercial/analysis/app/models/analysis/analysis.rb

```diff
@@ -30,12 +30,23 @@ class Analysis < ApplicationRecord
 
     validate :project_xor_phase_present
 
+    def inputs
+      scope = Idea.published
+      if phase_id
+        scope.in_phase(phase_id)
+      elsif project.timeline? # must be ideation
+        scope.where(project_id: project_id, creation_phase: nil)
+      elsif analysis.project.continuous?
+        scope.where(project_id: analysis.project_id)
```

Comment:
I don't get where the `analysis` is coming from here. Intuitively, I would have expected something like:
```suggestion
      elsif project.continuous?
        scope.where(project_id: project_id)
```

Reply:
That's totally right, and making it obvious we're missing some coverage on this, will fix and add